python/ray/serve/tests/unit/test_schema.py

```diff
@@ -266,6 +267,32 @@ def test_ge_zero_deployment_schema(self):
                 DeploymentSchema.parse_obj(deployment_schema)
             deployment_schema[field] = None
 
+    def test_validate_max_queued_requests(self):
+        # Ensure ValidationError is raised when max_queued_requests is not -1 or > 1.
+
+        deployment_schema = self.get_minimal_deployment_schema()
```

Comment:
Non-blocker: Thanks for adding those tests. I think we can test default as well as something that's not an int (e.g. "foobar", 1.23, "123") 🙃

Reply:
good idea